Actions/Github-Helper.psm1

```diff
@@ -407,6 +468,39 @@ function GetReleases {
     }
 }
 
+function GetLatestRelease {
+    Param(
+        [string] $token,
+        [string] $api_url = $ENV:GITHUB_API_URL,
+        [string] $repository = $ENV:GITHUB_REPOSITORY,
+        [string] $ref = $ENV:GITHUB_REFNAME
+    )
+    
+    Write-Host "Getting the latest release from $api_url/repos/$repository/releases/latest - branch $ref"
+    # Get all releases from GitHub, sorted by SemVer tag
+    # If any release tag is not a valid SemVer tag, use default GitHub sorting and issue a warning
+    # Default github sorting will return the latest historically created release as the latest release - not the highest version
+    $releases = GetReleases -token $token -api_url $api_url -repository $repository
+
+    # Get Latest release
+    $latestRelease = $releases | Where-Object { -not ($_.prerelease -or $_.draft) } | Select-Object -First 1
+    if ($ref -like 'release/*') {
+        try {
+            # If release branch, get the latest release from that the release branch
+            # This is given by the latest release with the same major.minor as the release branch
+            $semVerObj = SemVerStrToSemVerObj -semVerStr $ref.SubString(8) -allowMajorMinorOnly
```

Comment:
Consider something like `$ref.Replace("release/","")` instead of `$ref.SubString(8)`. I find that a bit more readable personally